configs/distill/mmcls/ofd/README.md

```diff
@@ -0,0 +1,160 @@
+# A Comprehensive Overhaul of Feature Distillation
+
+## Abstract
+
```

Comment:
DONE,

ps. since mmclassification has not the right wrn we needed, we have show vanilla model details, as a result,

split results and models table into two.